flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/delegation/PlannerBase.scala

```diff
@@ -148,6 +149,15 @@ abstract class PlannerBase(
     translateToPlan(execNodes)
   }
 
+  protected def overrideEnvParallelism(): Unit = {
```

Comment:
What about moving the implementation to `mergeParameters()`, so that `StreamPlanner` and `BatchPlanner` don't need to call this method again.

Reply:
`mergeParameters()` is not called by explain. So `StreamPlanner` and `BatchPlanner` still need to call this method.
And I think `overrideEnvParallelism` is near `translateToPlan` that is better.